.vscode/settings.json

```diff
@@ -42,8 +42,6 @@
     ".pre-commit-config.yaml": true,
     ".secrets.baseline": true,
     ".docforge": true,
-    ".github": true,
```

Comment:
Do you want to delete these lines or is this by accident?

Reply:
I intentionally removed these lines.
Excluding the .github folder makes it cumbersome to edit the github workflows etc. that's why I wanted to have it visible in vscode